sdk/core/Azure.Core.Experimental/src/Primitives/BinaryData.cs

```diff
@@ -13,58 +13,80 @@
 namespace Azure.Core
 {
     /// <summary>
-    /// Abstraction for a payload of binary data.
+    /// A lightweight abstraction for a payload of UTF-8 encoded bytes. This type integrates with <see cref="ObjectSerializer"/>
```

Comment:
This is a great comment!

Reply:
I think the comment is incorrect. Binary data can represent bytes that are not UTF8 encoded or not text at all.

Reply:
We should rename it to something like EncodedText otherwise.

Reply:
Yes, technically it can be non-textual data, but it is assumed to be UTF-8 encoded bytes. I can remove "UTF-8" from here though.

Reply:
I would remove it from description and have it on ToString.